weeks/week02/section02.md

```diff
@@ -0,0 +1,196 @@
+# Section 2: Tuples
+
+In this section, we will explore tuples, another core data structure in Python.
+
+`Tuple` is a type of data structure that is similar to lists. The main difference between the two is that tuples are `immutable`, meaning they cannot be changed after creating. Here a list of characteristics about Tuples:
+
+- Tuples are `immutables`,can't be changed
+- Used to store collections of heterogeneous data (any type of data: string, boolean, etc)
+- Rounded by parenthesis `()`
+- Only two built-in method:
+  - `count()`: Returns the number of times a specified value occurs in a tuple
+  - `index()`: Searches the tuple for a specified value and returns the position of where it was found
+- Uses `less memory` and is `faster` to access to elements when comparing with `lists`
+
+
+## References:
+
+1. [Tuples and Sequences - Python Documentation](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences)
+2. [Lists vs Tuples in Python - Real Python](https://realpython.com/python-tuples/)
+3. [Python Tuples - W3Schools](https://www.w3schools.com/python/python_tuples.asp)
+4. [Python Tuples - GeeksForGeeks](https://www.geeksforgeeks.org/python-tuples/)
+
+---
+
+
+## 2.1 Creating
+
+Tuples are useful in test automation for storing `immutable` collections of test data, such as constant values or configuration parameters. Tuples can be created values separated by commas, using parentheses `()` or the `tuple()` constructor.
```

Comment:
Would it sound better `Tuples can be created using values separated by commas`?

Reply:
Yes, can be